content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -148,8 +189,19 @@ spec:
   # to this address.
   - address: <IPofCluster2IngressGateway>
     ports:
-      http1: 15443 # Do not change this port value
-      tcp2: 15443 # Do not change this port value
+      http1: 15443
+      tcp2: 15443
+{{< /text >}}
+
+__Note__: If your Kubernetes cluster is running in an environment that does not
```

Comment:
please use `>` instead of an underscored note.  If it is very noteworthy, try using an icon instead.  See: https://preliminary.istio.io/about/contribute/writing-a-new-topic/#adding-icons-emojis